libs/orders/src/lib/components/order-data-provider/order-data-provider.ts

```diff
@@ -99,23 +98,43 @@ export const update = (
           draft.unshift(...draft.splice(index, 1));
         }
       } else if (newer) {
-        draft.unshift({ node, cursor: '', __typename: 'OrderEdge' });
+        const { marketId, ...order } = node;
+        draft.unshift({
+          node: {
+            ...order,
+            market: {
+              __typename: 'Market',
+              id: marketId,
+            },
+            __typename: 'Order',
+          },
+          cursor: '',
+          __typename: 'OrderEdge',
+        });
       }
     });
   });
 };
 
+// #TODO Order name is in conflict with interface defines in use-order-submit
```

Comment:
Will do. I want to look one more time on type names and refactor them. After all changes we made it may be easier to use global types from @vegaprotocol/types than use types generated for per query